### PR TITLE
#579 limit on_range_change calls

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
call on range change only when xaxis.* is present in the relayout event

### CODE REVIEW

File: gui/src/components/Taipy/Chart.tsx [hunk 1]
```diff
@@ -413,29 +413,31 @@ const Chart = (props: ChartProp) => {
...
-                        backCols,
-                        dataKey.current,
-                        getDecimatorsPayload(
-                            config.decimators,
-                            plotRef.current,
-                            config.modes,
-                            config.columns,
-                            config.traces,
-                            eventData
+            if (Object.keys(eventData).some(k => k.startsWith("xaxis."))) {
```
Comment (FabienLelaquais) [inline gui/src/components/Taipy/Chart.tsx:416]: What about "|| ...yaxis...?

**Review by FabienLelaquais — APPROVED** [review]
👌

**Review by dinhlongviolin1 — APPROVED** [review]

File: gui/src/components/Taipy/Chart.tsx (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/src/components/Taipy/Chart.tsx:416]: I tested the different use cases, and I could find a case when yaxis is present and not xaxis ...

Comment (FabienLelaquais) [reply gui/src/components/Taipy/Chart.tsx:416]: Ok then... fingers crossed

**Pull request merged**